execution_engine/src/lib.rs

```diff
@@ -2,8 +2,8 @@
 
 #![doc(html_root_url = "https://docs.rs/casper-execution-engine/3.0.0")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```

Comment:
Can we put the Casper logo here?  The source code is owned by Casper Assocation, not Labs.